### ISSUE TITLE
How to take derivatives with respect to a vector

### ISSUE THREAD

**Issue by andymancodes** [pytorch/pytorch#12439]
Hi,

In Pytorch, much of the autograd functionality is under wraps using the variable abstraction.

I just wanted to use autograd to implement linear regression.

I have a vector [x1 x2 x3], I want to take the derivative with respect to a loss function.

How can I do that?

Since, Pytorch is a drop-in replacement for numpy as well, I think I can use it like that along with the autograd feature.

Cheers!

**Comment by soumith** [comment]
Hi @andymancodes . If you followed the issue templates, it must've told you that Questions and Help are available at https://discuss.pytorch.org
Looking forward to your post there instead :)
